helm/fluent-logshipping-app-chart/templates/fluentbit/service.yaml

```
@@ -6,6 +6,11 @@ metadata:
   labels:
     app: {{ .Values.fluentbit.name }}
   annotations:
+    {{- if .Values.giantswarm.supporting }}
```

```suggestion
    {{- if .Values.giantswarm.monitoring.enabled }}
```

Thanks for the change. I think it's more readable like this. But if we want to use this as a coding convention it will need to be documented.

---

That's better. I would make a change as soon.